fission/main.go

```diff
@@ -124,6 +218,35 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "get-env-variables",
```

Comment:
How slow is it to fetch the url every time? And btw, if we switch to using the apiserver proxy, we won't have to talk to the api to get the url.